ui/js/langs.js

```diff
@@ -0,0 +1,196 @@
+/*
```

Comment:
We're working to reduce top-level files, though there's not an obvious place for this. I guess `js/util` for now?